appservice/src/connectToGitHub.ts

```diff
@@ -64,13 +67,18 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
     };
 
     try {
-        const connectingToGithub: string = localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo. This may take several minutes...', client.fullName);
-        const connectedToGithub: string = localize('ConnectedToGithub', '"{0}" has been connected to the GitHub repo.', client.fullName);
-        await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: connectingToGithub }, async (): Promise<void> => {
-            ext.outputChannel.appendLine(connectingToGithub);
-            await client.updateSourceControl(siteSourceControl);
-            vscode.window.showInformationMessage(connectedToGithub);
-            ext.outputChannel.appendLine(connectedToGithub);
+        const deploying: string = localize('deploying', 'Deploying to "{0}"... Check output window for status.', client.fullName);
+        const deploymentComplete: string = localize('deploymentComplete', 'Deployment to "{0}" completed.', client.fullName);
+        const kuduClient: KuduClient = await getKuduClient(client);
+        await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: deploying }, async (): Promise<void> => {
```

Comment:
The title should not be "deploying" until after you change the source control

Reply:
In this case, changing the source control and deploying are the same command.  So I'm not really sure how to distinguish between the two actions.

Reply:
Ah - well then I prefer the "connecting to github" message over the "deploy" message. But it should still say "Check output window for status." instead of " This may take several minutes..."